sio-digital-team/active-safe
# New overview

[trello card](https://trello.com/c/6j3Nly2E/120-code-new-sport-overview)


**Changes:**

- [x] Rebuilt new overview section
  

**How to test:**

- view in mobile, tablet, and desktop
- click on tabs, hotspots should change accordingly and vice versa
  

**Todo:**
